ROOTFS/etc/sage-utils/dhcp-pxe/nfs/etc/rc.local

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+#initializes sd card on rpi
+
+SDDIR="/dev/mmcblk0"
+SDPARTDIR="/dev/mmcblk0p1"
+
+if [ -b "$SDDIR" ]; then
+  echo "SD Card found continuing..." | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` > /etc/rc.local.logs
+  mkdir -p /media/scratch/ | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
+  if [ -b "$SDPARTDIR" ]; then
+        echo "Mounting SD CARD Scratch Space" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
+        mkfs -t ext4 /dev/mmcblk0p1 | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
```

Comment:
um you dont want to do mkfs on every mount.  you only want to do the mkfs after formatting the partitions.

fyi you can also use the `mkfs.ext4` command. don't matter.

```
root@70ffebe896aa:/# ls -la /sbin/mkfs.ext4
lrwxrwxrwx 1 root root 6 Jan 22  2020 /sbin/mkfs.ext4 -> mke2fs
```